src/main/java/htsjdk/samtools/cram/build/CramContainerHeaderIterator.java

```diff
@@ -22,8 +22,19 @@ public CramContainerHeaderIterator(final InputStream inputStream) {
       super(inputStream);
     }
 
+    /**
+     * Consume the entirety of the next container from the stream,
+     * but retain only the header.
+     *
+     * @see CramContainerIterator#containerFromStream(Version, CountingInputStream)
+     *
+     * @param cramVersion the expected CRAM version of the stream
+     * @param countingStream the {@link CountingInputStream} to read from
+     * @return The next Container's header from the stream, returned as a Container.
+     */
+    @Override
```

Comment:
This tripped me up!

Hopefully this explicit Override and Javadoc will save some future grief.